share/completions/strace.fish

```diff
@@ -1,24 +1,74 @@
 # Startup
+complete -c strace -s E -l env -d 'Remove var from the inherited list of environment variables before passing it on to the command'
 complete -c strace -s p -l attach -xa '(__fish_complete_pids)'
 complete -c strace -s u -l user -xa '(__fish_complete_users)'
+complete -c strace -l argv0 -d 'Set argv[0] of the command being executed to name'
 
 # Tracing
 complete -c strace -s b -l detach-on -d 'Detach when the specified syscall is reached'
+complete -c strace -s D -l daemonize -d 'Run tracer process as a grandchild, not as the parent of the tracee'
+complete -c strace -s DD -l daemonize -d "Run  tracer  process as tracee's grandchild in a separate process group."
+complete -c strace -s DDD -l daemonize -d "Run tracer process as tracee's grandchild in a separate session"
+complete -c strace -s f -l follow-forks
+complete -c strace -l output-separately
+complete -c strace -s ff -d 'Combine the effects of --follow-forks and --output-separately options'
```

Comment:
The "options" at the end is a bit redundant, I'd just make it "Like --follow-forks --output-separately".

It should also be an old-style `-o` option.